tests/unit/archive_base_test.py

```diff
@@ -405,6 +405,407 @@ def test_check_missing_files(datapackage, baseline_resources, new_resources, suc
     assert validation_result.success == success
 
 
+@pytest.mark.parametrize(
+    "baseline_resources,new_resources,success",
+    [
+        (
+            # Test when file size changes by > threshold for any one file
+            [
+                Resource(
+                    name="resource0",
```

Comment:
It'll make the test cases way easier to understand if we make this more compact with a factory function that only takes the varying parameters. I think it shouldn't be much more than this:

```
def _resource_w_size(name, size):
    return Resource(
        name=name,
        path=f"https://www.example.com/{name}",
        ...
        bytes=size
    )
```